### DISCUSSION TITLE
Issues with loading hard and easy chartings of songs, issues with loading normal in freeplay

### DISCUSSION THREAD

**Discussion by PlasmaOJ15** [KadeArchive/Kade-Engine#518] [Q&A]
The game will crash and say Null Object whenever I try to play the easy or hard version of a song I added. In freeplay, it tries to play  easy when its on normal difficulty. I have no clue what is happening

**Comment by PlasmaOJ15** [comment]
Now it works fine on debug mode, but normally loading it doesnt????

**Comment by PlasmaOJ15** [comment]
So resaving it in debug fixed that i guess. But still have no clue why easy and hard mode isnt working

**Comment by puyoxyz** [comment]
did you actually create the `songname-hard.json` and `songname-easy.json`

**Reply by PlasmaOJ15** [reply]
yes, but it says it is missing

**Reply by Kade-github** [reply]
did you compile with those already in the game? otherwise, it won't detect them as existing.

**Reply by PlasmaOJ15** [reply]
yes
